src/components/DayPickerNavigation.jsx

```diff
@@ -177,8 +188,9 @@ function DayPickerNavigation({
           ]),
         )}
         aria-label={phrases.jumpToNextMonth}
-        onClick={onNextMonthClick}
+        onClick={hasNext ? onNextMonthClick : () => {}}
```

Comment:
Same comments apply here about using undefined instead of a thunk

Reply:
Curious: I guess there's an advantage by using undefined instead of a thunk ?

Reply:
Setting it to undefined is akin to not having that prop set at all, which means no unnecessary click handler. Otherwise you're adding a click handlers to things that don't need them, only to call an empty function upon invocation.